app/frontend/src/components/forms/FormSubmission.vue

```diff
@@ -19,6 +19,28 @@
         </v-col>
         <v-spacer />
         <v-col class="text-sm-right" cols="12" sm="6">
+          <span v-if="canViewAllSubmissions()">
+            <v-tooltip bottom>
+              <template #activator="{ on, attrs }">
+                <router-link
+                  :to="{ name: 'FormSubmissions', query: { f: form.id } }"
+                >
+                  <v-btn
+                    class="mx-1"
+                    color="primary"
+                    text
+                    v-bind="attrs"
+                    v-on="on"
+                  >
+                    <v-icon class="mr-1">content_copy</v-icon>
+                    <span>Submissions</span>
```

Comment:
This button should be an icon only - it should not have text built-into the button as that goes against our top-right button navigation paradigm.